hw/ip/dcd/data/dcd_testplan.hjson

```diff
@@ -0,0 +1,33 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "dcd"
```

Comment:
Isnt this getting renamed to adctrl? Why not rename first, then generate the DV tb and then integrate into the top? That way there is less churn.

Reply:
yeah i definitely jumped the gun here.... i think at this point i'm just going to try and get it in as dcd and do a big rename PR to follow on..

you're right though, for the other module `rbox`, it makes sense to rename it to sys_ctrl first.

@esshiu 
Maybe it makes sense for you to work on renaming `rbox` first? My suggestion yesterday wasn't the best, since I kind of neglected all the downstream things I'm adding.